Added constellations

ARMargolis opened the pull request:
Added Sentinel, Landsat and BlackSky constellations

ARMargolis committed:
Added constellations

Added Sentinel, Landsat and BlackSky constellations

DylanBartels commented on src/satellite_orbit_predictor/app.py:
```diff
@@ -8,6 +8,14 @@
 
 constellations = {
     "orbital-sidekick": {"GHOST-1":56195, "GHOST-2":56197, "GHOST-3":56958, "GHOST-4":59133, "GHOST-5":59130},
+    "sentinel": {'SENTINEL-1A': 39634, 'SENTINEL-2A': 40697, 'SENTINEL-3A': 41335, 'SENTINEL-1B': 41456, 
```
I think these are actually different constellations/stapi products:
Sentinel-1 (1A, 1B, 1C): Radar imaging for all-weather, day-and-night observations (C-band SAR).
Sentinel-2 (2A, 2B, 2C): Multispectral optical imaging, mainly for land monitoring (vegetation, soil, water, etc.).
Sentinel-3 (3A, 3B): Ocean and land monitoring — sea surface temperature, ocean color, altimetry, etc.
Sentinel-5P: Atmospheric monitoring (pollutants like NO₂, CO, O₃), precursor to Sentinel-5.
Sentinel-6: Measures sea surface height with high accuracy (continuation of the Jason missions).

ARMargolis replied:
I'll make the Sentinel-1, -2, and -3 series different constellations. Sentinel-5 and -6 are just single satellites, so they don't really test the use case.

ARMargolis committed:
Divided Sentinel Constellation

I divided the Sentinel constellation into -1, -2, and -3 series. I removed Sentinel-5 and -6 since they're only one each.
